change/@ni-nimble-components-c5bf4c01-bf3e-4fa0-b4fd-0bbc8b4a1762.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm not sure how rigorous we have been or want to be about keeping specs up to date, but `packages/nimble-components/src/table/specs/README.md` has a section under **API** for *Events* which could link to this new document too.

Reply:
I realized we didn't have `action-menu-toggle` documented there (only `action-menu-beforetoggle`), so I've included that as well.